.rubocop_todo.yml

```diff
@@ -104,14 +104,15 @@ Lint/UnreachableLoop:
   Exclude:
     - 'test/unit/image_s3_test.rb'
 
-# Offense count: 9
+# Offense count: 10
 # Configuration parameters: CheckForMethodsWithNoSideEffects.
 Lint/Void:
   Exclude:
     - 'app/classes/mo_paginator.rb'
     - 'app/classes/query/modules/high_level_queries.rb'
     - 'app/classes/query/modules/nested_queries.rb'
     - 'app/classes/query/modules/sequence_operators.rb'
+    - 'app/controllers/description_controller_helpers.rb'
```

Comment:
This represents a new offense I'd like to get rid of. 
RuboCop says
>Line #658 – Warning: Lint/Void: Operator - used in void context.
    `new_groups - old_groups.each do |group`

I think this is a bug in RuboCop, as it's not flagging similar Line 662:
`old_groups - new_groups.each do |group|`

Reply:
I think this exposes a bug in our code (in addition to a bug in RuboCop).
Here is the whole method. (app/controllers/description_controller_helpers.rb lines 657-666).
```ruby
  # Throw up some flash notices to reassure user that we did in fact make the
  # changes they wanted us to make.
  def flash_description_changes(old_groups, new_groups, type)
    new_groups - old_groups.each do |group|
      name = group_name(group)
      flash_notice(:"runtime_description_added_#{type}".t(name: name))
    end
    old_groups - new_groups.each do |group|
      name = group_name(group)
      flash_notice(:"runtime_description_removed_#{type}".t(name: name))
    end
  end
```
I think we need to parenthesize `new_groups - old_groups` (and later, `old_groups - new_groups`).
As currently written, I think the block
```ruby
    new_groups - old_groups.each do |group|
      name = group_name(group)
      flash_notice(:"runtime_description_added_#{type}".t(name: name))
    end
```
will flash each member of `old_groups`, and return `new_groups - old_groups`.  See example below.
But I suspect that what was intended was to flash the members of ` new_groups - old_groups`.
Example:
```ruby
> a_e = %w[a b c d e]
 => ["a", "b", "c", "d", "e"] 
> d_e = %w[d e]
 => ["d", "e"] 
> a_e - d_e.each { |l| puts l}
d
e
 => ["a", "b", "c"] 
```
(RuboCop is buggy in saying that `-` is invalid in this context.)

Reply:
You've convinced me!  I'm sure this is something I wrote... but I don't typically use the each construct.  I bet this was originally a for loop and it just got converted hastily.  Obviously not a very important piece of code since no one has complained!